releasenotes/notes/release-1.0-758fc917e40d7eb3.yaml

```
@@ -0,0 +1,81 @@
+---
+prelude: >
+  The Datadog APM Python team is happy to announce the release of v1.0.0 of
+  ddtrace.
+
+
+  The first commit to the library was made on June 20, 2016. Nearly sixty minor
+  releases later, the library has grown immensely. The library now includes over
+  60 integrations for libraries. Moreover, the library has expanded out from
+  tracing to support the Continuous Profiler, CI Visibility, and Application
+  Security.
+
+
+  Our motivations for this major version release were:
+
+    - Clarify and simplify the public API.
+    - Introduce a versioning policy.
+    - Provide users a painless upgrade path.
+
+
+  .. note::
+  Before upgrading to v1.0.0, we recommend running tests with `warning filters <https://docs.python.org/3.8/library/warnings.html#warning-filter>`_ set to turning warnings into exceptions. For example::
+
+    $ PYTHONWARNINGS=error python tests.py
+    $ pytest -W tests.py
```

we need to limit to only `ddtrace` warnings.

@mabdinur to confirm, but I think we need this:

```suggestion
    $ PYTHONWARNINGS="error::DeprecationWarning:ddtrace[.*]" python tests.py
    $ pytest -W="error::DeprecationWarning:ddtrace[.*]" tests.py
```

---

Yup that looks right